cmake/shaderc.pc.in

```diff
@@ -0,0 +1,12 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=${prefix}
+libdir=${prefix}/@CMAKE_INSTALL_LIBDIR@
+includedir=${prefix}/@CMAKE_INSTALL_INCLUDEDIR@
+
+Name: shaderc
+Description: Tools for Vulkan shaders.
```

Comment:
This description is too broad, overlapping too much with SPIRV-Tools.

The Shaderc github repo summary is "A collection of tools, libraries and tests for Vulkan shader compilation."

I suggest for the purpose of these pkg.config files, shorten to: "Tools and libraries for Vulkan shader compilation."

Reply:
will do